eth_bloom/crypto.py

```diff
@@ -0,0 +1,13 @@
+from Crypto.Hash import keccak
```

Comment:
Would you be interested in implementing this over in https://github.com/carver/eth-hash/blob/master/eth_hash/backends/pycryptodome.py ?

I'm going to talk to @pipermerriam about getting that repository moved into the ethereum org, and start depending on it everywhere. Some repositories that will be affected:

- eth-utils
- web3.py
- py-evm

Reply:
* py-trie